### ISSUE TITLE
Entrega código

### ISSUE THREAD

**Issue by ADRYEP** [mmasias/idsw2#52]
Buenos días profesor,
Viendo la confusión que ha generado entre los alumnos de la materia (yo incluido) quiero preguntarle por las fechas puestas en el repo, coloca que el día 25, hoy, se cierra la entrega de los cambios hechos no sólo en el código sino también en los diagramas cuando en clase habíamos acordado que dicho cierre se realizaría el 28 y mismo día comenzarán las presentaciones. Entiendo que estos cambios los haya publicado el día 13 de marzo (antes de vacaciones de semana santa) pero como sabrá estos cambios no son notificados.
Por lo tanto, quisiera saber si existe la posibilidad de mantener la fecha de entrega de código para el 28.

**Comment by mmasias** [comment]
Sí, mantenemos lo dicho en clase. Esto es, se aceptan pull request hasta el día anterior a la presentación (el 28).
